NEWS.md

```diff
@@ -140,72 +134,53 @@ Standard library changes
     other result in dense output. Moreover, constructing 3-arg similar matrices of special
     "sparse" matrices of (nonstatic) matrices now fails for the lack of `zero(::Type{Matrix{T}})`.
 
-#### Markdown
-
 #### Printf
+
 * Now uses `textwidth` for formatting `%s` and `%c` widths ([#41085]).
 
 #### Profile
+
 * CPU profiling now records sample metadata including thread and task. `Profile.print()` has a new `groupby` kwarg that allows
   grouping by thread, task, or nested thread/task, task/thread, and `threads` and `tasks` kwargs to allow filtering.
   Further, percent utilization is now reported as a total or per-thread, based on whether the thread is idle or not at
   each sample. `Profile.fetch()` by default strips out the new metadata to ensure backwards compatibility with external
-  profiling data consumers, but can be included with the `include_meta` kwarg. ([#41742])
+  profiling data consumers, but can be included with the `include_meta` kwarg ([#41742]).
```

Comment:
This can be removed as it's no longer the case (I couldn't do a suggestion for some reason)
```
`Profile.fetch()` by default strips out the new metadata to ensure backwards compatibility with external
  profiling data consumers, but can be included with the `include_meta` kwarg
```

Reply:
Should we just invert the meaning, i.e. by default it includes new metadata, which can be removed by specifying `include_meta=false`?

Reply:
Ok yeah, that works